internal/api/impl/auth/auth.go

```diff
@@ -56,15 +58,69 @@ func (e *Endpoint) auth(ctx echo.Context) error {
 	if !crypto.ComparePasswords(usr.Spec.Password, body.Password) {
 		return shared.HandleBadRequestError("wrong login or password ")
 	}
-	token, err := e.jwt.SignedToken(body.Login)
+	accessToken, err := e.accessToken(ctx, body.Login)
 	if err != nil {
-		logrus.WithError(err).Errorf("unable to generate the JWT token")
-		return shared.InternalError
+		return err
 	}
-	jwtHeaderPayloadCookie, signatureCookie := e.jwt.CreateJWTCookies(token)
-	ctx.SetCookie(jwtHeaderPayloadCookie)
-	ctx.SetCookie(signatureCookie)
+	refreshToken, err := e.refreshToken(ctx, body.Login)
+	if err != nil {
+		return err
+	}
+
 	return ctx.JSON(http.StatusOK, api.AuthResponse{
```

Comment:
Don´t you need to change the UI part because of this change ? To at least have a working dev env?

Reply:
yup I need to work on that to complete this PR !